using-timescaledb/data-retention.md

```diff
@@ -20,6 +27,41 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
+### Data Retention and Continuous Aggregates
+
+The function `drop_chunks` can be used to control data retention for
+the continuous aggregate as well, but if you have a continuous
+aggregate defined on the table you need to provide a value for the
+`cascade_to_materialization`. Not giving a value when there is a
+continuous aggregate defined on a hypertable will generate an error.
+
+Providing the value `FALSE` to `cascade_to_materialization` will only
+remove chunks from the hypertable and not cascade to dropping chunks
+on the continuous aggregate.
+
+It is not possible to remove chunks from the hypertable without
+allowing the corresponding chunks to be removed from the continuous
+aggregate, so it is necessary to alter the continuous aggregate and
+set `ignore_invalidation_older_than` to the range to remove.
```

Comment:
This is also super-odd. Why isn't this set automatically when you do drop_chunks without cascade_to_materialization?

Again, just double-check that this is actually the case because UX here is very strange.

Reply:
Here I am trying to drop chunks from `conditions` without having set `ignore_invalidations_older_than`, and then setting `ignore_invalidations_older_than` for the aggregate and repeating the operation.

```
postgres-#   FROM drop_chunks(INTERVAL '30 days', 'conditions',
postgres(#                    cascade_to_materializations => FALSE);
ERROR:  older_than must be greater than the timescaledb.ignore_invalidation_older_than parameter of public.conditions_summary
postgres=# ALTER VIEW conditions_summary SET (
postgres(#    timescaledb.ignore_invalidation_older_than = '30 days'
postgres(# );
ALTER VIEW
postgres=# SELECT COUNT(*) AS dropped_chunks
postgres-#   FROM drop_chunks(INTERVAL '30 days', 'conditions',
postgres(#                    cascade_to_materializations => FALSE);
NOTICE:  making sure all invalidations for public.conditions_summary have been processed prior to dropping chunks
 dropped_chunks 
----------------
           1080
(1 row)
```

Reply:
This statement is confusing.  Are you saying that you can't remove chunks from the hypertable without removing the dependency of the cagg on those dropped chunks?  I think you need to state that.  Your current statement implies that the user would be trying to remove the actual chunks from the cagg table (i.e. dropping chunks on `conditions_materialized_view` table, or whatever it's called).

Reply:
Fixed wording.